statistics/handle/dump.go

```diff
@@ -96,18 +96,20 @@ type jsonColumn struct {
 	TotColSize        int64           `json:"tot_col_size"`
 	LastUpdateVersion uint64          `json:"last_update_version"`
 	Correlation       float64         `json:"correlation"`
+	StatsVer          *int64            `json:"stats_ver"`
```

Comment:
Why use a pointer here?

Reply:
It could be empty if the stats is old version.